src/App.css

```diff
@@ -1,62 +1,65 @@
-* {
```

Comment:
You're doing a lot of CSS - try to keep it fairly basic for now and stick to functionality and getting those tickets mastered